### PR TITLE
DOC: Fix capitalization among headings in documentation files (#32550)

### PR DESCRIPTION
**Pull request by themien**
Following #32978 and #32944.

Headers updated for the following files:
```
- [ ] doc/source/getting_started/comparison/comparison_with_stata.rst
- [ ] doc/source/getting_started/intro_tutorials/01_table_oriented.rst
- [ ] doc/source/getting_started/tutorials.rst
- [ ] doc/source/reference/arrays.rst
- [ ] doc/source/reference/window.rst
- [ ] doc/source/user_guide/boolean.rst
```

### CODE REVIEW

**Comment by MomIsBestFriend** [comment]
Can you please merge master?

**Comment by themien** [comment]
HI @MomIsBestFriend , sorry I am quite new to git. Could you please expand on what I should do?

**Comment by MomIsBestFriend** [comment]
> HI @MomIsBestFriend , sorry I am quite new to git. Could you please expand on what I should do?

@themien Sure!

If your working environment is set up with the instructions found in the [contributing guide](https://dev.pandas.io/docs/development/contributing.html), you can merge "master" by running:

```
git checkout bug_32550_part_3
git fetch upstream
git merge upstream/master
```

And then, push your code again:

```
git push origin bug_32550_part_3
```

**Comment by themien** [comment]
@MomIsBestFriend great! Thank you

**Comment by jreback** [comment]
looks ok. @datapythonista @jorisvandenbossche

File: doc/source/getting_started/comparison/comparison_with_stata.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {{ header }}
 
-Comparison with Stata
+Comparison with STATA
```
Comment (datapythonista) [inline doc/source/getting_started/comparison/comparison_with_stata.rst]: Can you revert this one please? Stata is capitalized `Stata` in their website, so let's use that.

**Review by datapythonista — COMMENTED** [review]
Nice, just one comment

File: doc/source/getting_started/comparison/comparison_with_stata.rst (hunk 1 above)
Comment (themien) [reply doc/source/getting_started/comparison/comparison_with_stata.rst]: Sure! Thanks for pointing that out.

**Comment by datapythonista** [comment]
Thanks @themien, nice clean up

**Pull request merged**